src/cs/production/Flecs/World.cs

```diff
@@ -14,11 +14,46 @@ namespace Flecs;
 [PublicAPI]
 public unsafe class World
 {
+    // Relationships
+    public Entity EcsIsA => new Entity(this, pinvoke_EcsIsA());
```

Comment:
Is it better to have these cached or just return a new struct instance everytime?